usaspending_api/api_contracts/contracts/v2/idvs/funding.md

```diff
@@ -33,7 +33,7 @@ List File C financial data for an IDV award's descendant contracts
                     Program activity name, program activity code
                 + `reporting_agency_name`
                 + `reporting_fiscal_date`
-                    Reporting fiscal year, reporting fiscal period
+                    Reporting fiscal year, reporting fiscal month
```

Comment:
are we able to sort by `gross_outlay_amount` and `disaster_emergency_fund_code `?

Reply:
What are you talking about?  They're right there!  😛

Reply:
@kbard where? I don't see them in the sort enum list either

Reply:
![image](https://user-images.githubusercontent.com/43386699/87468222-307f7c80-c5e7-11ea-9ee1-aeffa913b2f7.png)

Reply:
![image](https://user-images.githubusercontent.com/22781949/87483038-f49ce500-c5ef-11ea-90b4-940ddba5c644.png)

Reply:
We can chalk that up to github's improved U/I hiding commits